Reflection/Interfaces.simba

```diff
@@ -130,186 +130,21 @@ begin
-    R_SPELL_BLOOD_RUSH,
-    R_SPELL_BLOOD_BLITZ,
-    R_SPELL_BLOOD_BURST,
-    R_SPELL_BLOOD_BARRAGE,
-    R_SPELL_SMOKE_RUSH,
-    R_SPELL_SMOKE_BLITZ,
-    R_SPELL_SMOKE_BURST,
-    R_SPELL_SMOKE_BARRAGE,
-    R_SPELL_SHADOW_RUSH,
-    R_SPELL_SHADOW_BLITZ,
-    R_SPELL_SHADOW_BURST,
-    R_SPELL_SHADOW_BARRAGE,
-    R_SPELL_PADDEWWA_TELEPORT,
-    R_SPELL_SENNTISTEN_TELEPORT,
-    R_SPELL_KHARYRLL_TELEPORT,
-    R_SPELL_LASSAR_TELEPORT,
-    R_SPELL_DAREEYAK_TELEPORT,
-    R_SPELL_CARRALLANGER_TELEPORT,
-    R_SPELL_ANNAKARL_TELEPORT,
-    R_SPELL_GHORROCK_TELEPORT,
-    R_SPELL_EDGEVILLE_HOME_TELEPORT,
-    R_SPELL_LUNAR_HOME_TELEPORT,
-    R_SPELL_BAKE_PIE,
-    R_SPELL_CURE_PLANT,
-    R_SPELL_MONSTER_EXAMINE,
-    R_SPELL_NPC_CONTACT,
-    R_SPELL_CURE_OTHER,
-    R_SPELL_HUMIDIFY,
-    R_SPELL_MOONCLAN_TELEPORT,
-    R_SPELL_TELE_GROUP_MOONCLAN,
-    R_SPELL_CURE_ME,
-    R_SPELL_HUNTER_KIT,
-    R_SPELL_WATERBIRTH_TELEPORT,
-    R_SPELL_TELE_GROUP_WATERBIRTH,
-    R_SPELL_CURE_GROUP,
-    R_SPELL_STAT_SPY,
-    R_SPELL_BARBARIAN_TELEPORT,
-    R_SPELL_TELE_GROUP_BARBARIAN,
-    R_SPELL_SUPERGLASS_MAKE,
-    R_SPELL_TAN_LEATHER,
-    R_SPELL_KHAZARD_TELEPORT,
-    R_SPELL_TELE_GROUP_KHAZARD,
-    R_SPELL_DREAM,
-    R_SPELL_STRING_JEWELLERY,
-    R_SPELL_STAT_RESTORE_POT_SHARE,
-    R_SPELL_MAGIC_IMBUE,
-    R_SPELL_FERTILE_SOIL,
-    R_SPELL_BOOST_POTION_SHARE,
-    R_SPELL_FISHING_GUILD_TELEPORT,
-    R_SPELL_TELE_GROUP_FISHING_GUILD,
-    R_SPELL_PLANK_MAKE,
-    R_SPELL_CATHERBY_TELEPORT,
-    R_SPELL_TELE_GROUP_CATHERBY,
-    R_SPELL_RECHARGE_DRAGONSTONE,
-    R_SPELL_ICE_PLATEAU_TELEPORT,
-    R_SPELL_TELE_GROUP_ICE_PLATEAU,
-    R_SPELL_ENERGY_TRANSFER,
-    R_SPELL_HEAL_OTHER,
-    R_SPELL_VENGEANCE_OTHER,
-    R_SPELL_VENGEANCE,
-    R_SPELL_HEAL_GROUP,
-    R_SPELL_SPELLBOOK_SWAP,
-    R_SPELL_GEOMANCY,
-    R_SPELL_SPIN_FLAX,
-    R_SPELL_OURANIA_TELEPORT,
-    R_SPELL_ARCEUUS_HOME_TELEPORT,
-    R_SPELL_REANIMATE_GOBLIN,
-    R_SPELL_UNKNOWN_1,
-    R_SPELL_REANIMATE_MONKEY,
-    R_SPELL_REANIMATE_IMP,
-    R_SPELL_REANIMATE_MINOTAUR,
-    R_SPELL_UNKNOWN_2,
-    R_SPELL_REANIMATE_SCORPION,
-    R_SPELL_REANIMATE_BEAR,
-    R_SPELL_REANIMATE_UNICORN,
-    R_SPELL_REANIMATE_DOG,
-    R_SPELL_UNKNOWN_3,
-    R_SPELL_REANIMATE_CHAOS_DRUID,
-    R_SPELL_UNKNOWN_4,
-    R_SPELL_REANIMATE_GIANT,
-    R_SPELL_UNKNOWN_5,
-    R_SPELL_REANIMATE_OGRE,
-    R_SPELL_REANIMATE_ELF,
-    R_SPELL_REANIMATE_TROLL,
-    R_SPELL_UNKNOWN_6,
-    R_SPELL_REANIMATE_HORROR,
-    R_SPELL_REANIMATE_KALPHITE,
-    R_SPELL_UNKNOWN_7,
-    R_SPELL_REANIMATE_DAGANNOTH,
-    R_SPELL_REANIMATE_BLOODVELD,
-    R_SPELL_UNKNOWN_8,
-    R_SPELL_REANIMATE_TZHAAR,
-    R_SPELL_UNKNOWN_9,
-    R_SPELL_REANIMATE_DEMON,
-    R_SPELL_REANIMATE_AVIANSIE,
-    R_SPELL_UNKNOWN_10,
-    R_SPELL_UNKNOWN_11,
-    R_SPELL_REANIMATE_ABYSSAL,
-    R_SPELL_UNKNOWN_12,
-    R_SPELL_REANIMATE_DRAGON,
-    R_SPELL_BATTLEFRONT_TELEPORT,
-    R_SPELL_UNKNOWN_13
-  ];
```

Comment:
Why did we remove all of these?

Reply:
@Brandon-T Converted it all into the different spellbooks, therefore these are essentially useless as this is all the spellbooks combined.

Reply:
got it. thanks!